lib/active_model/serializer.rb

```diff
@@ -36,6 +36,10 @@ def self.attribute(attr, options = {})
       end unless method_defined?(key)
     end
 
+    def self.column_names
+      to_s.sub("Serializer", "").constantize.column_names
```

Comment:
I don't know if this is a valid requirement or not, but instead of relying on the serializer name and doing string manipulation, you could just use `resource.class`